modules/stochastic_tools/src/state_sim/common/StateSimBitset.C

```diff
@@ -0,0 +1,156 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "StateSimBitset.h"
```

Comment:
There is a bitset class in C++11. Maybe you should use that as the container for this class?